src/hera/v1/cron_workflow.py

```diff
@@ -0,0 +1,56 @@
+"""The implementation of a Hera cron workflow for Argo-based cron workflows"""
+
+from typing import Optional
+
+from argo.workflows.client import (
+    V1alpha1CronWorkflow,
+    V1alpha1CronWorkflowSpec,
+)
+from hera.v1.cron_workflow_service import CronWorkflowService
+from hera.v1.workflow_service import WorkflowService
+from hera.v1.workflow import Workflow
+
+
+class CronWorkflow(Workflow):
```

Comment:
I would add some simple tests to ensure that the right fields are set on this class!

Reply:
Will do as soon as we agree on implementation

Reply:
@flaviuvadan added test_cwf_valid_field_set